Multisig/UI/Transaction/ExecuteTransaction/AmountAndValuePiece.swift

```diff
@@ -21,8 +21,19 @@ class AmountAndValuePiece: UINibView {
         setTextAlignment(.right)
     }
 
-    func setAmount(_ value: String?) {
-        amountLabel.text = value
+    func setAmount(_ value: String?, sponsored: Bool = false) {
```

Comment:
Maybe it would be better if we use more generalized name rather than sponsored, something that covers all possible causes why we strikethrough that label

Reply:
it's over optimisation at this point. We can revisit naming later, but now `sponsored` suits quite well here for now.